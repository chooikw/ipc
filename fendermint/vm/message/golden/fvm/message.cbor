8a1b126923327a33e01455014d89139a6956cf0eff9f10f98ef6b2331be4ae014b0098d6afc8dd9ff78a8c011bb969cd3d854c218f5100720c82482aa174dcff80732dfdaf409c1bea746c0fa2d46b1d4051004092b29472652110a350d7b229295d8e1bffe9ee70ba70ea44451a4eab4ab7