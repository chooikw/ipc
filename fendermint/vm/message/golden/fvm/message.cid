0171a0e40220a0c9994025a46636152c2dd2e50a71d876e0a4e66852431f95d65afc6b7c6997