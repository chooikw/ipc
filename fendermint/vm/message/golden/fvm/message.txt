Message { version: 1326630264942878740, from: Address("f010094217122647108376"), to: Address("f1jwerhgtjk3hq5747cd4y55vsgmn6jlqbduzoiva"), sequence: 13360435433684345231, value: TokenAmount(151596941512652562527.424789902313603228), method_num: 18440532317176916548, params: RawBytes { 1a4eab4ab7 }, gas_limit: 16894246916585712413, gas_fee_cap: TokenAmount(0.0), gas_premium: TokenAmount(85832289101661751580.490929022670822798) }